tests/send_CAPI_command.py

```diff
@@ -16,14 +16,63 @@ class CAPIReply(Enum):
     ERROR = "status,ERROR"
 
 
+class tlv:
+    """Representation of an 1905.1 TLV."""
+
+    def __init__(self, type_: int, length: int, value: str):
+        """A TLV has a type, a length and a value.
+
+        Parameters
+        ----------
+        type_ : int
+            The TLV type.
+
+        length : int
+            The length of the TLV. This must correspond to the length of `value`.
+
+        value : str
+            The TLV value, as a string. It must be formatted according to the UCC rules, i.e. with
+            curly braces and hexadecimal numbers.
+        """
+        self.type = type_
+        self.length = length
+        self.value = value
+
+    def format(self, tlv_num: int = 0) -> str:
+        """Format the TLV for the dev_send_1905 CAPI command.
+
+        Parameters
+        ----------
+        tlv_num : int
+            The TLV counter. In the CAPI command, if there are multiple TLVs, they must be numbered
+            starting from 1. `tlv_num` is that number. If there is only a single TLV, set it to
+            zero.
+
+        Returns
+        -------
+        str
+            The string representation of the TLV according to CAPI definition.
+        """
+        if tlv_num:
+            tlv_num = str(tlv_num)
```

Comment:
I think it's bad practice to change the type of a variable, especially when you explicitly mark it as int in the method signature. 
I would prefer defining a tlv_num_str variable.

Reply:
Good point. Done.